playground/resources/demo-ui/src/pages/home.tsx

```diff
@@ -4,61 +4,126 @@ import {
   MonitorRequestProps,
   RequestRecord,
 } from '../components/monitor-request'
-import { Box, Typography, styled } from '@mui/material'
+import { Box, Typography, styled, Tabs, Tab } from '@mui/material'
+import { TabContext, TabList, TabPanel } from '@mui/lab'
 import {
   GracefulError,
   GracefulErrorProps,
   useGracefulRequest,
 } from '@fluxninja-tools/graceful-js'
-import { api } from '../api'
+import { api, RequestSpec } from '../api'
+
 
 export const HomePage: FC = () => {
+
+  const [value, setValue] = useState('1');
+
+  const handleChange = (event: React.ChangeEvent<{}>, newValue: string) => {
+    setValue(newValue);
+  };
+  const reqSpec: RequestSpec = {
+    method: 'POST',git
+    endpoint: '/rate-limit',
+    userType: 'Guest',
+    userId: 'Vu',
+  };
+
+  // Request to rate-limit endpoint with Guest user
   const {
     refetch,
     isError,
-    requestRecord: crawlerRequestRecord,
-    isLoading: isLoadingCrawler,
-  } = useGracefulRequestForRateLimit('Crawler')
+    requestRecord: requestRecord,
+    isLoading: isLoadingRequest,
+  } = makeRequestToEndpoint(reqSpec)
+
+  // Request to workload-prioritization endpoint with Guest user
+  reqSpec.endpoint = '/workload-prioritization'
   const {
     refetch: refetchSubscriber,
     isError: isErrorSubscriber,
     requestRecord: subscriberRequestRecord,
     isLoading: isLoadingSubscriber,
-  } = useGracefulRequestForRateLimit('Subscriber')
+  } = makeRequestToEndpoint(reqSpec)
+
+  // Request to workload-prioritization endpoint with Subscriber user
+  reqSpec.userType = 'Subscriber'
+  const {
+    refetch: refetchSubscriber2,
+    isError: isErrorSubscriber2,
+    requestRecord: subscriberRequestRecord2,
+    isLoading: isLoadingSubscriber2,
+  } = makeRequestToEndpoint(reqSpec)
```

Comment:
@karansohi follow react hook convenstion. If it is a hook add use infront of that function name `useRequestToEndpoint`